README.md

```diff
@@ -1,9 +1,7 @@
 # SATRE Specification Repository: Technical Document for the Standard Architecture for Trusted Research Environments Project
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-
-[![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
-
+[![All Contributors](https://img.shields.io/badge/all_contributors-9-orange.svg?style=flat-square)](#contributors-)
```

Comment:
```suggestion

[![All Contributors](https://img.shields.io/badge/all_contributors-9-orange.svg?style=flat-square)](#contributors-)

```

Reply:
The bot/CLI will revert this change whenever it is used... 🤔

Reply:
Added ignore tags for the badge in 001d008